src/views/virtualmachines/details/tabs/metrics/NetworkCharts/hook/utils/utils.ts

```diff
@@ -0,0 +1,4 @@
+import { PrometheusResponse, PrometheusValue } from '@openshift-console/dynamic-plugin-sdk';
```

Comment:
This could also be useful for the Overview navitem for the monitoring tab, can you please extract this function to shared?

Reply:
Sure